README.md

```diff
@@ -28,11 +28,9 @@ PyTorch Frame is a deep learning extension for [PyTorch](https://pytorch.org/),
 
 PyTorch Frame democratizes deep learning research for tabular data, catering to both novices and experts alike. Our goals are:
 
-1. **Facilitate Deep Learning for Tabular Data:** Historically, tree-based models (e.g., XGBoost) excelled at tabular learning but had notable limitations, such as integration difficulties with downstream models such as GNNs, and handling complex column types (e.g., text, sequences, images). Deep tabular models are promising to resolve the limitations.
+1. **Facilitate Deep Learning for Tabular Data:** Historically, tree-based models (e.g., XGBoost) excelled at tabular learning but had notable limitations, such as integration difficulties with downstream models such as GNNs, and handling complex column types (e.g., text, sequences, images). Deep tabular models are promising to resolve the limitations. We enhance PyTorch Frame to handle diverse column types, like timestamps, multicategorical values, texts, sequences and embeddings.
```

Comment:
Historically, tree-based models (e.g., XGBoost) excelled at tabular learning but had notable limitations, such as integration difficulties with downstream models, and handling complex column types, such as texts, sequences, and embeddings. Deep tabular models are promising to resolve the limitations. We aim to facilitate deep learning research on tabular data by modularizing its implementation and supporting the diverse column types.